litmus-portal/frontend/src/components/TuneWorkflow/styles.ts

```diff
@@ -0,0 +1,54 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: 'rgba(255, 255, 255, 0.6)',
+    height: '100%',
+    width: '61.25rem',
```

Comment:
This is a very minor change to make it 80% of the whole screen width(covered in #1808 ) to fit the design requirement, its not an urgent need but could be modified later.

Reply:
Done.